extensions/web/11_blob_url.js

```diff
@@ -33,7 +33,7 @@
     const url = core.opSync(
       "op_file_create_object_url",
       blob.type,
-      blob[_byteSequence],
+      getParts(blob),
```

Comment:
@lucacasonato I created util fn for getting the BlobReferences as a array of strings
Not sure how it's suppose to be called

Feedback?